metadata_tools/hosts/GO_0xxx/gcp_geometry_startup.sh

```
@@ -0,0 +1,10 @@
+apt-get update -y
+apt-get install -y python3 python3-pip python3-venv git
+cd
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)